### DISCUSSION TITLE
RustDesk freeze if not installed on client

### DISCUSSION THREAD

**Discussion by adispy** [rustdesk/rustdesk#3836] [General]
Hi,
Like it sais in the title, the screen just freeze if I do not install RustDesk on the client side. The OS is Windows on both sides, the connecting one and the client one. I tested with both UAC disabled and enabled but the results are the same.
The encryption is not forced on the Docker container (-k_) in order to keep it simple.

Most of the times I cannot install the client and I need it just for instant access.

Any ideas?

Thanks,

**Comment by rustdesk** [comment]
https://github.com/rustdesk/rustdesk/issues/70

**Reply by adispy** [reply]
I understand but RustDesk client was launched using a Domain Admin account and UAC disabled on that specific server, so no restrictions.
